src/models/SqlOutputContentProvider.ts

```diff
@@ -535,4 +534,51 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
         }
         return path.join(os.tmpdir(), columnName + '_' + String(Math.floor( Date.now() / 1000)) + String(process.pid) + '.' + linkType);
     }
+
+    /**
+     * Returns wether or not a result pane with the same URI exists
+     * @param The string value of a Uri.
+     * @return Existence boolean
+     * public for testing purposes
+     */
+    public doesResultPaneExist(resultsUri: string): boolean {
+        let resultPaneURIMatch = vscode.workspace.textDocuments.find(tDoc => tDoc.uri.toString() === resultsUri);
```

Comment:
please use `this._vscodeWrapper.textDocuments()` instead of direct references so it's easier to mock.